DirectProgramming/C++SYCL_FPGA/Tutorials/Features/multi_aocx/device_link/README.md

```diff
@@ -0,0 +1,241 @@
+# Using the flows test 'device_link'
+
+This FPGA tutorial demonstrates the device link flow
+
+| Optimized for                     | Description
+|:---                               |:---
+| OS                                | Linux* Ubuntu* 18.04/20.04 <br> RHEL*/CentOS* 8 <br> SUSE* 15 <br> Windows* 10
+| Hardware                          | Intel&reg; Programmable Acceleration Card (PAC) with Intel Arria&reg; 10 GX FPGA <br> Intel&reg; FPGA Programmable Acceleration Card (PAC) D5005 (with Intel Stratix&reg; 10 SX) <br> Intel&reg; FPGA 3rd party / custom platforms with oneAPI support <br> **Note**: Intel&reg; FPGA PAC hardware is only compatible with Ubuntu 18.04*
+| Software                          | Intel® oneAPI DPC++/C++ Compiler
+| What you will learn               | How to use the `device_link` flow tests to parallelize the compiles of multiple images
+| Time to complete                  | 30 minutes
+
+> **Note**: Even though the Intel DPC++/C++ OneAPI compiler is enough to compile for emulation, generating reports and generating RTL, there are extra software requirements for the simulation flow and FPGA compiles.
+>
+> For using the simulator flow, one of the following simulators must be installed and accessible through your PATH:
+> - Questa*-Intel® FPGA Edition
+> - Questa*-Intel® FPGA Starter Edition
+> - ModelSim® SE
+>
+> When using the hardware compile flow, Intel® Quartus® Prime Pro Edition must be installed and accessible through your PATH.
+
+## Purpose
+
+The purpose of this tutorial is to describe the `device_link` flow and to show how it can be used to parallelize the individual compiles when users don't need to restructure their code. 
+
+### Using the `device_link` flow
+`device_link` flow compile each hardware image separately and links all compiled images into a single fat binary.
```

Comment:
Check the other samples to see what language we use (eg: "We can compile" vs "You can compile")
```suggestion
In this flow, you can compile each hardware image separately and link all compiled images into a single fat binary. Here are some advantages of this flow:
```